src/problem-domain-analysis/classes-objects-and-structure.tex

```diff
@@ -1,3 +1,59 @@
 % textidote: ignore begin
 \section{Classes, Objects and Structure}\label{sec:classes-objects-and-structure}
 % textidote: ignore end
+
+In this section the classes, objects and structure of the proposed solution will be explored, this
+will help get a more detail understanding of how the system should function in relation to the problem at hand.
+
+
+% textidote: ignore begin
+\subsection{Classes and Objects}\label{subsec:classes-and-objects}
+% textidote: ignore end
+To understand the classes that are relevant for the issues Nova café is facing,
+a general description of the classes is given in~\ref{tab:class-table}.
+
+% textidote: ignore begin
+\begin{table}[h]
+    \begin{tabular} { m{2.5cm} m{10cm} }
```

Comment:
Forcing all column widths leaves strange margins, especially since you didn't use `\centering`
<img width="1164" alt="image" src="https://github.com/user-attachments/assets/5dc70b45-797a-47ad-95ad-1a2822f7c861">
You can use `tabularx` and the `X` column
```suggestion
    \begin{tabular} { m{2.5cm} X }
```

Reply:
latex does not compile when i use either X or x

Reply:
A Google search like `latex X column type` and/or a closer look at my comment would have clarified this:
```latex
\usepackage{tabularx}
```

Reply:
Could not get it to work with begin{tabularx} { m{2.5cm} X }. Used \centering instead.